src/App.jsx

```diff
@@ -1,10 +1,106 @@
+import axios from 'axios'
 import './App.css';
+import {useState} from "react";
+import globe from "./assets/spinning-globe.jpg"
+import worldmap from "./assets/world_map.png"
+import regionColorNames from "./helpers/regionColorHelper.js";
+import sortPopulationHelper from "./helpers/sortPopulationHelper.js";
 
 function App() {
 
+    const [countries, setCountries] = useState("")
+    const [countryName, setCountryName] = useState("")
+    const [countryData, setCountryData] = useState("")
```

Comment:
Hier geldt hetzelfde voor, maar dan `undefined`
```suggestion
    const [countryData, setCountryData] = useState();
```